### PR TITLE
Add health certificate validity endpoint

### PR DESCRIPTION
**Pull request by vermakhushboo**
## What does this PR do?

Get status of certificate for a domain to check whether it is still valid or expired.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/api/health.php [hunk 1]
```diff
@@ -389,6 +392,56 @@
...
+    ->groups(['api', 'health'])
+    ->label('scope', 'health.read')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
+    ->label('sdk.namespace', 'health')
+    ->label('sdk.method', 'getCertificate')
+    ->label('sdk.description', '/docs/references/health/get-certificate.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_HEALTH_CERTIFICATE)
+    ->param('domain', null, new Multiple([new Domain(), new PublicDomain()]), Multiple::TYPE_STRING, 'Domain name')
```
Comment (christyjacob4) [inline app/controllers/api/health.php:406]: Should the default value be `null`? How does the endpoint behave when no`domain` is passed ?

File: composer.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
         "utopia-php/cli": "0.15.*",
         "utopia-php/config": "0.2.*",
         "utopia-php/database": "0.45.*",
-        "utopia-php/domains": "0.3.*",
+        "utopia-php/domains": "0.5.*",
```
Comment (christyjacob4) [inline composer.json:53]: is this update expected? what did we include in the new version?

File: tests/e2e/Services/Health/HealthCustomServerTest.php [hunk 3]
```diff
@@ -424,4 +424,67 @@ public function testStorageAntiVirusSuccess(): array
...
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), []);
+
+        $this->assertEquals(200, $response['headers']['status-code']);
+
+        /**
+         * Test for FAILURE
+         */
+        $response = $this->client->call(Client::METHOD_GET, '/health/certificate?domain=localhost', array_merge([
+            'content-type' => 'application/json',
```
Comment (christyjacob4) [inline tests/e2e/Services/Health/HealthCustomServerTest.php:468]: Can we also add a test for an empty domain?

File: src/Appwrite/Extend/Exception.php [hunk 4]
```diff
@@ -235,6 +235,8 @@ class Exception extends \Exception
 
     /** Health */
     public const QUEUE_SIZE_EXCEEDED                 = 'queue_size_exceeded';
```
Comment (christyjacob4) [inline src/Appwrite/Extend/Exception.php:237]: While we are at it can we also update the key here to `HEALTH_QUEUE_SIZE_EXCEEDED`

File: app/controllers/api/health.php (hunk 1 above)
Comment (vermakhushboo) [reply app/controllers/api/health.php:406]: When no domain is passed, the validator throws a 400 error. Added a test.

**Review by christyjacob4 — APPROVED** [review]
